2020-04-30 20:33:37,283 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:37,284 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:37,285 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:38,390 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:38,391 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:38,393 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:39,495 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:39,496 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:39,497 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:40,601 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:40,603 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:40,604 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:41,710 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:41,711 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:41,713 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:42,816 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:42,817 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:42,817 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:43,919 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:43,919 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:43,921 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:45,023 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:45,024 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:45,026 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:46,132 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:46,133 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:46,135 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:47,238 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:47,239 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:47,241 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:48,343 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:48,350 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:48,352 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:49,454 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:49,455 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:49,457 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:50,559 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:50,562 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:50,564 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:51,667 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:51,668 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:51,679 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:52,781 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:52,781 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:52,781 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:53,883 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:53,883 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:53,885 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:54,987 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:54,988 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:54,989 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:56,090 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:56,091 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:56,092 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:57,194 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:57,194 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:57,194 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:58,303 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:58,303 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:58,305 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:33:59,407 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:33:59,409 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:33:59,414 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:00,518 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:00,519 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:00,521 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:01,623 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:01,624 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:01,625 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:02,728 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:02,729 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:02,731 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:03,833 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:03,833 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:03,835 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:04,937 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:04,937 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:04,938 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:06,040 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:06,040 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:06,042 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:07,144 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:07,145 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:07,146 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:08,249 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:08,249 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:08,251 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:09,353 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:09,353 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:09,355 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:10,457 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:10,457 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:10,458 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:11,564 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:11,565 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:11,567 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:12,670 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:12,671 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:12,673 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:13,775 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:13,779 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:13,780 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:14,883 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:14,883 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:14,885 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:15,988 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:15,989 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:15,990 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:17,092 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:17,093 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:17,095 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:18,196 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:18,197 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:18,198 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:19,300 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:19,301 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:19,302 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:20,404 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:20,404 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:20,406 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:21,510 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:21,510 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:21,511 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:22,630 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:22,650 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:22,697 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:23,799 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:23,800 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:23,801 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:24,909 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:24,915 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:24,927 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:26,028 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:26,030 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:26,032 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:27,136 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:27,136 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:27,136 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:28,239 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:28,240 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:28,244 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:29,345 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:29,345 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:29,346 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:30,448 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:30,449 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:30,452 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:31,562 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:31,563 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:31,573 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:32,684 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:32,685 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:32,687 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:33,791 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:33,792 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:33,793 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:34,897 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:34,897 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:34,898 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:36,000 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:36,000 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:36,002 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:37,103 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:37,104 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:37,108 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:38,209 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:38,211 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:38,212 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:39,326 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:39,327 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:39,328 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:40,432 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:40,432 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:40,433 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:41,534 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:41,535 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:41,536 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:42,638 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:42,639 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:42,640 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:43,742 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:43,742 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:43,743 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:44,845 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:44,846 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:44,847 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:45,949 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:45,950 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:45,952 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:47,059 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:47,059 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:47,060 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:48,162 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:48,162 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:48,163 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:49,264 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:49,264 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:49,265 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:50,366 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:50,367 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:50,375 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:51,477 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:51,477 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:51,479 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:52,580 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:52,590 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:52,591 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:53,692 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:53,693 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:53,693 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:54,794 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:54,795 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:54,795 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:55,897 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:55,898 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:55,898 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:56,999 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:56,999 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:57,000 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:58,101 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:58,101 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:58,102 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:34:59,202 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:34:59,203 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:34:59,203 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:00,306 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:00,307 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:00,308 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:01,409 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:01,409 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:01,411 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:02,512 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:02,512 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:02,513 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:03,614 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:03,615 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:03,616 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:04,718 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:04,718 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:04,720 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:05,823 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:05,823 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:05,825 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:06,926 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:06,927 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:06,928 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
2020-04-30 20:35:08,030 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1154] - Opening socket connection to server localhost/127.0.0.1:2181.
2020-04-30 20:35:08,030 [myid:localhost:2181] - INFO  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1156] - SASL config status: Will not attempt to authenticate using SASL (unknown error)
2020-04-30 20:35:08,040 [myid:localhost:2181] - WARN  [pt.tecnico.sauron.silo.SiloServerApp.main(localhost:2181):ClientCnxn$SendThread@1272] - Session 0x0 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException.
java.net.ConnectException: Ligação recusada
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)
